e2e/pages/app/badge/badge.testpage.data.ts

```diff
@@ -0,0 +1,37 @@
+import { BadgeHorizontalPosition, BadgeSize, BadgeVerticalPosition } from '../../../../src/directives/badge';
```

Comment:
If I recall I think the e2e tests are setup so this should be importing from `@ux-aspects/ux-aspects`. If that doesn't work then we should use this path but include `/index` on the end and Angular can sometimes be a bit awkward when it comes to import paths excluding the `index` part of the path